Tests/bootstrap.php

```diff
@@ -0,0 +1,6 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+require_once __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
This will only work in ezplatform/ezstudio setup, that is not used for travis tests where they are run isolated.
see: https://github.com/ezsystems/ezpublish-kernel/blob/master/bootstrap.php
https://github.com/ezsystems/PlatformUIBundle/blob/master/Tests/bootstrap.php